app/Activity.php

```diff
@@ -46,19 +46,19 @@ public function isAllDay()
      *
      * @return DateTime
      */
-    public function getStart()
+    public function getSortingDate()
     {
-        return new DateTime($this->datetimestart);
+        return new DateTime($this->sorting_date);
     }
 
     /**
      * Get the end time
```

Comment:
mooie refactor, dit is precies waarom comments in code een code smell is, ze rotten weg door aanpassingen ;P